nilearn/_utils/compat.py

```diff
@@ -58,3 +62,33 @@ def md5_hash(string):
         m = hashlib.md5()
         m.update(string)
         return m.hexdigest()
+
+
+def get_affine(img):
+    """A function to return affine upon nibabel version.
+
+    nibabel > 2.0.0 returns img.affine otherwise img.get_affine()
+    """
+    if isinstance(img, _basestring):
+        img = nibabel.load(img)
+
+    if LooseVersion(nibabel.__version__) > LooseVersion('2.0.0'):
+        affine = img.affine
```

Comment:
checking the version at each call is not the best practice. You should put a condition on the declaration of the function at module level:
```python
if LooseVersion(nibabel.__version__) > LooseVersion('2.0.0'):
    def get_affine(img):
        return img.affine
else:
    def get_affine(img):
        return img.get_affine()
```